modules/nw-aws-nlb-new-cluster.adoc

```diff
@@ -19,7 +19,7 @@ Create an Ingress Controller backed by an AWS NLB on a new cluster.
 +
 [source,terminal]
 ----
-$ ./openshift-install create manifests --dir=<installation_directory>
+$ ./openshift-install create manifests --dir=<installation_directory> <1>
```

Comment:
@jboxman Can you check this update please?

Reply:
Your change aligns with the usage elsewhere in the install docs.